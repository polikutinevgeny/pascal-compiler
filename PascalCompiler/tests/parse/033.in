var
   n: integer;

begin
   n := 3;
   while n < 0 do
end.