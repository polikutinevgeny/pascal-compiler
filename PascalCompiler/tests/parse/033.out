program 
  integer: (integer = (built-in));
  real: (real = (built-in));
  char: (char = (built-in));
  n: n : (integer = (built-in)) ;
├─Assignment statement
│ ├─Designator
│ │ └─n
│ └─3
└─while
  ├─Less
  │ ├─Designator
  │ │ └─n
  │ └─0
  └─
